Src/Controllers/LogCaptureControllers.lua

```diff
@@ -0,0 +1,55 @@
+local ScriptContext = game:GetService("ScriptContext")
+
+local Console = require(script.Parent.Parent.Packages.Console)
+local Signal = require(script.Parent.Parent.Packages.Signal)
+
+local Network = require(script.Parent.Parent.Network.Client)
+
+local MESSAGE_REPORTER_DELAY = 60
+
+local LogCaptureControllers = {}
+
+LogCaptureControllers.Priority = 0
+LogCaptureControllers.Reporter = Console.new(`{script.Name}`)
+
+LogCaptureControllers.MessageQueue = {}
+
+LogCaptureControllers.MessageQueueUpdated = Signal.new()
```

Comment:
I actually adopted what you said @ethndotsh lol - things will just be signals internally, just so I don't have so many cyclic dependency issues.. ugh..

But yeah these signals _SHOULDN'T_ be used by the end user :smile: